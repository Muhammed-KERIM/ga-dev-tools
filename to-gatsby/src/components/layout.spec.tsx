import * as React from "react"
import * as renderer from "react-test-renderer"

import Layout from "./layout"

describe("Layout", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Layout>Content</Layout>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
