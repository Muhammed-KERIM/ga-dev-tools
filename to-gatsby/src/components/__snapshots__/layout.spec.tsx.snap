// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <nav
    className="makeStyles-nav-1"
  >
    <h5
      className="MuiTypography-root MuiTypography-h5"
    >
      Demos & Tools
    </h5>
    <ol>
      <li>
        <a
          href="/autotrack/"
        >
          Autotrack
        </a>
      </li>
      <li>
        <a
          href="/account-explorer/"
        >
          Account Explorer
        </a>
      </li>
    </ol>
  </nav>,
  <main
    className="makeStyles-main-2"
  >
    Content
  </main>,
]
`;
